certora/specs/misc/EVC_Prop25_ResetsTransientStorage.spec

```diff
@@ -0,0 +1,41 @@
+/**
+ * Verification of:
+ *   All the storage variables declared in the TransientStorage contract must return to the default value after the top-level EVC call.
+ **/
+
+import "../utils/IsMustRevertFunction.spec";
+
+methods {
+    function getRawExecutionContext() external returns (uint256) envfree;
+    function getExecutionContextDefault() external returns (uint256) envfree;
+    function areAccountStatusChecksEmpty() external returns (bool) envfree;
+    function areVaultStatusChecksEmpty() external returns (bool) envfree;
+    function getCurrentOnBehalfOfAccount(address) external returns (address,bool) envfree;
+}
+
+/**
+ * Verify that after any function call of call depth zero, both account and
```

Comment:
the concept of call depth no longer exists. now we simply have a boolean `checksDeferred` flag

Reply:
I have removed mention of call depth from the `certora` directory